config/settings/test.yml

```diff
@@ -89,4 +89,3 @@ veteran_readiness_and_employment:
   base_url: https://fake_url.com
   credentials: fake_auth
   auth_endpoint: https://fake_url.com/auth/oauth/token
-  ch_31_endpoint: /fake/endpoint
```

Comment:
what was the motivation for taking these out of the config?

Reply:
Hey @erluti thank you for looking into this.

I suppose the motivation was to reduce complexity.

It was overkill to include that path in the config and I got my wires crossed with ops. I had sent them the `credentials`, `base_url`, and `auth_endpoint` but not the `ch_31_endpoint`.

So in testing, I realized that when we made the request that uses the `ch_31_endpoint`, the string wasn't there.

Instead of opening a request with ops to add the string to credstash, we have this PR. I'm in sort of a time crunch.

I will include some conversations and PR's as reference below.

https://dsva.slack.com/archives/CBU0KDSB1/p1606492837200400

https://dsva.slack.com/archives/CLY6Q69RV/p1605822014289500?thread_ts=1605210607.227700&cid=CLY6Q69RV

https://github.com/department-of-veterans-affairs/devops/pull/8060